### PR TITLE
adding overview and routing slides

### PR DESCRIPTION
**Pull request by turbobeast**

### CODE REVIEW

File: src/content/introduction.md [hunk 1]
```diff
@@ -8,3 +8,91 @@
...
+- 20 min Intro/Slides
+- 1h 40 min Live Coding
+
+---
+
+## Curriculum
+
+- Routing (with React Router V4)
+- Server-Side Rendering (with express.js)
+- TBD
```
Comment (rbrander) [inline src/content/introduction.md]: I think we should either add Lazy Loading here or remove the bullet all together

File: src/content/introduction.md [hunk 2]
```diff
@@ -8,3 +8,91 @@
...
+- Navigation within a single-page application (without page reload)
+- Deep-link (address-bar)
+- Browser history
+
+---
+
+## React Router V4
+
+- declarative API
+- browser history API by default
```
Comment (rbrander) [inline src/content/introduction.md]: To be consistent, we should uppercase the first letter of each bullet point

**Review by rbrander — CHANGES_REQUESTED** [review]
Overall, good content, just a couple of simple changes

**Comment by turbobeast** [comment]
I addressed your changes. I also had some local changes I hadn't pushed yet (messy, messy). Have a look. You'll have to rebase your branch.

**Review by rbrander — APPROVED** [review]
LGTM :shipit:

**Pull request merged**